### ISSUE TITLE
Muti-speaker training for synthesizer and the vocoder

### ISSUE THREAD

**Issue by dipjyoti92** [CorentinJ/Real-Time-Voice-Cloning#173]
I would like to thank @CorentinJ for this awesome repo. It really is very helpful. I would be very much glad if you could answer the following question which are confusing to me.

According to the repo (https://github.com/CorentinJ/Real-Time-Voice-Cloning/wiki/Training), for synthesizer and the vocoder, it is mentioned that train-clean-100, train-clean-360 data sets are used which have 1000+ speakers. 

Synthesizer has speaker embedding (Figure 15 of the thesis) where multi-speaker training can be done. So, the training for the Synthesizer was done with 1000+ speakers. Right?

However, in the vocoder training (uses fatchord/WaveRNN repo), multi-speaker training is not included. I am wondering how the vocoder training was done (i.e., using which speakers)?

Thanks in advance.

**Comment by CorentinJ** [comment]
The vocoder is not conditioned on any speaker identity, rather it is trained on all the data available from all the speakers.

**Comment by dipjyoti92** [comment]
Thanks for the quick answer. However, Synthesizer is conditioned on multiple speakers, Is that right?

**Comment by CorentinJ** [comment]
The synthesizer is conditioned on speakers with speaker embeddings, yes.

**Comment by dipjyoti92** [comment]
Hi @CorentinJ Thank you for your replies. I have another two questions if you could kindly reply:

(i) Is the WaveRNN vocoder trained with GTA aligned mel-spec from tacotron2 or mel-specs generated from natural speech?

(ii) I see in the WaveRNN hyper-parameters, some of the default parameters are not used as it is used in fatchord/WaveRNN's repo such as 
voc_upsample_factors = (5, 5, 11) ==> voc_upsample_factors = (5, 5, 8) 
voc_mode = 'MOL'==> voc_mode = 'RAW'   etc,

Is there any particular reason? Did you finetune yourself depending on the performance of the output samples?

**Comment by yakimovsa** [comment]
I noticed that solo-speaker trained WaveRNN vocoder outputs voice of much higher quality than one trained on the whole multi-speaker dataset. You must have speaker ID labels to train solo-speaker vocoder.

**Comment by blue-fish** [comment]
Closing this issue due to inactivity. @dipjyoti92 if you would like an answer to your additional questions, try asking in #364 .
